File: C Programming/Assignments/Assignment 7/Q3.c
//3. Write a program to check whether a given year is a leap year.
#include<stdio.h>
void cleap(int*);

void main ()
{
    int a;
    printf("Enter a year :");
    scanf("%d",&a);
    cleap(&a);
}

void cleap(int *pt)
{
    if(*pt % 4 == 0 && *pt % 100 != 0)
    printf("the year is leap");
    else
    printf("the year is not leap");
}

File: Java/C TO JAVA/Assignment 1/Q3.java
class Q3
{
public static void main(String args[]){
	int year=2001;
	if(year%4==0&&year%100!=0||year%400==0){
		System.out.printf("Ths year is leap");
		
	}else{
		System.out.printf("year is not leap");
	}
}
}